samples/esb/esb_monitor/sample.yaml

```diff
@@ -0,0 +1,58 @@
+sample:
+  name: ESB Monitor Sample
+tests:
+  sample.esb.monitor:
```

Comment:
I think we can add the `sample.esb.monitr.build:` case for the basic configuration, where all supported platform will be listed.